libs/api/domains/national-registry-x-road/src/lib/nationalRegistryXRoad.service.ts

```diff
@@ -41,6 +42,38 @@ export class NationalRegistryXRoadService {
     }))
   }
 
+  async getNationalRegistryChildGuardianship(
```

Comment:
This service is named `NationalRegistryXRoadService` so I would shorten the method name to
```suggestion
  async getChildGuardianship(
```